app/models/comment.rb

```diff
@@ -1,11 +1,9 @@
 class Comment < ApplicationRecord
-  belongs_to :author, class_name: 'User', foreign_key: 'author_id'
-  belongs_to :post, class_name: 'Post', foreign_key: 'post_id'
+  belongs_to :user, foreign_key: 'user_id'
+  belongs_to :post
```

Comment:
- [ ] For some reason I am not able to create a comment I suggest that you check your association and foreign_keys
<img width="1127" alt="Screen Shot 2023-06-29 at 18 17 29" src="https://github.com/karanJ2212/RailsBlog/assets/44745855/7958ac5d-11ad-42b4-b6b1-696dc8da3f68">

Reply:
resolved